test/test_jit.py

```diff
@@ -2798,6 +2798,18 @@ def fn(x):
         fn(x)
         fn(y)
 
+    def test_python_ivalue(self):
+        # Test if pure python object can be hold as IValue and conversion
```

Comment:
This test should test memory management: if IValue is the only holder of the object then it should not go dead. It should also not cause the python object to leak.